{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Not affected",
      "package_name": "numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Not affected",
      "package_name": "numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Will not fix",
      "package_name": "inkscape:flatpak/numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Will not fix",
      "package_name": "numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Will not fix",
      "package_name": "python27:2.7/numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Will not fix",
      "package_name": "python38:3.8/numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Will not fix",
      "package_name": "python39:3.9/numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Will not fix",
      "package_name": "numpy",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat OpenStack Platform 13 (Queens)",
      "fix_state": "Not affected",
      "package_name": "numpy",
      "cpe": "cpe:/a:redhat:openstack:13"
    },
    {
      "product_name": "Red Hat OpenStack Platform 16.1",
      "fix_state": "Not affected",
      "package_name": "numpy",
      "cpe": "cpe:/a:redhat:openstack:16.1"
    },
    {
      "product_name": "Red Hat OpenStack Platform 16.2",
      "fix_state": "Not affected",
      "package_name": "numpy",
      "cpe": "cpe:/a:redhat:openstack:16.2"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Not affected",
      "package_name": "python27-numpy",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Will not fix",
      "package_name": "rh-python38-numpy",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2021-05-11T00:00:00Z",
  "bugzilla": {
    "description": "numpy: incomplete string comparison in the numpy.core component",
    "id": "2035032",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2035032"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "5.3",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-20",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2021-34141",
  "document_distribution": "",
  "details": [
    "An incomplete string comparison in the numpy.core component in NumPy before 1.22.0 allows attackers to trigger slightly incorrect copying by constructing specific string objects. NOTE: the vendor states that this reported code behavior is \"completely harmless.\""
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2021-34141\nhttps://nvd.nist.gov/vuln/detail/CVE-2021-34141"
  ]
}